share/vss/vss.go

```diff
@@ -0,0 +1,579 @@
+// vss.go implements the verifiable secret sharing scheme by Pedersen recalled
+// in "Provably Secure Distributed Schnorr Signatures and a (t, n) Threshold
+// Scheme for Implicit Certificates"  by Stinson and Strobl.
+// The basic scheme is taken from Pedersen's paper: "Non-interactive and
+// information-theoretic secure verifiable secret sharing", in Crypto ’91, pages
+// 129–140, 1991.
+// At a higher level, you have one party called the Dealer who wishes to share a
```

Comment:
`At a higher level` is a comparative which doesn't make much sense here. Maybe write:

```
VSS enables a dealer to share a secret securely and verifiably with a list of verifiers where at most t-1 out of n participants (dealer and/or verifiers) are allowed to be malicious. The verifiability of the process prevents a malicious dealer from influencing the outcome to his advantage as each verifier can check the validity of the received share. The protocol has the following steps:
```